### PR TITLE
fixed camel case?

### PR DESCRIPTION
**Pull request by DerekL27**
## Why

[Ticket](https://github.com/Code-4-Community/hands-across-the-sea-frontend/issues/57)
[Ticket](https://github.com/Code-4-Community/hands-across-the-sea-frontend/issues/58)

## This PR

Fixes camel case? Not sure if this is what the ticket meant

### CODE REVIEW

**Comment by justinkonecny** [comment]
Screenshots?

Did you test this to make sure it doesn't break anything?

**Comment by granttebeau** [comment]
Screen Shot 2021-08-03 at 11 24 05 AM

Still have camel case here for the country and privilege columns, we'll need to make these normal case as well in the user directory

**Comment by DerekL27** [comment]
image

@justinkonecny 
@granttebeau 

should be fixed now

**Review by granttebeau — APPROVED** [review]
LGTM

**Review by maxsebso — CHANGES_REQUESTED** [review]
The problem we had last time was that the form item would now contain no camel case. This would mean the request would come back bad request because the backend would be expecting camel case. If you are sure that you tested this thoroughly I will approve. If it is not tested it can and will break the whole form.

**Comment by DerekL27** [comment]
> The problem we had last time was that the form item would now contain no camel case. This would mean the request would come back bad request because the backend would be expecting camel case. If you are sure that you tested this thoroughly I will approve. If it is not tested it can and will break the whole form.

Yeah you're right it did break the form. I changed it so that it takes the enum and formats it to look nice so now everything should be good. I tested it and it works

**Review by maxsebso — CHANGES_REQUESTED** [review]
Almost! Just one thing. Since you are using the function convertEnumToRegularText 3 times in 3 different files you should put it in utils folder and then import it when it needs to be used

**Comment by DerekL27** [comment]
> Almost! Just one thing. Since you are using the function convertEnumToRegularText 3 times in 3 different files you should put it in utils folder and then import it when it needs to be used

Smart

File: src/utils/helpers.tsx [hunk 1]
```diff
@@ -19,6 +19,13 @@ export const getOptionsFromEnum = (e: {
   ));
 };
 
+export const convertEnumToRegularText = (input: string) => {
```
Comment (rymaju) [inline src/utils/helpers.tsx:22]:
```suggestion
export const convertEnumToRegularText = (input: string): string => {
```

**Review by rymaju — APPROVED** [review]
Small nitpick ⛏️ otherwise LGTM

**Review by maxsebso — APPROVED** [review]
LGTM

**Pull request merged**